### PR TITLE
Allow blank cell in xml editor

### LINKED ISSUE #4599: Blank cell type not accepted in visual editor
**Describe the bug**
When trying to enter a blank cell type in visual editor it does not let you save but in XML editor and 7.9.3.1 it does work.

**To Reproduce**

1. Go to form definition 
2. Have visual editor on
3. Add a blank cell type 
5. Switch to XML editor
6. See you can save 

**Expected behavior**
You should be able to save a blank cell type without issue. 

**Screenshots**

https://github.com/specify/specify7/assets/150940840/25242413-b804-4362-a6f3-3218a1aebf61

Alternatively, please fill out the following information manually:

- OS: Windows 11
- Browser: Chrome
- Specify 7 Version: edge
- Database Name: lsumzmammals9823

### PR DESCRIPTION
**Pull request by CarolineDenis**
Fixes #4599

### Checklist

- [x] Self-review the PR after opening it to make sure the changes look good
      and self-explanatory (or properly documented)
- [x] Add relevant issue to release milestone

### Testing instructions
Go to form definition
Have visual editor on
Add a blank cell type
See you can save

### CODE REVIEW

File: specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts [hunk 1]
```diff
@@ -313,6 +315,15 @@ const labelSpec = f.store(() =>
   })
 );
 
+const blankSpec = f.store(() =>
```
Comment (CarolineDenis) [inline specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts:318]: @maxpatiiuk, is this necessary or is using emptySpec sufficient?

Comment (maxpatiiuk) [reply specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts:318]: if the cell is blank, then it's not supposed to have any attributes, right?
so yeah, emptySpec is the right choice at the moment

though, when developing a visual editor, we would have to consider how empty cells and unknown cells should be treated

**Review by maxpatiiuk — APPROVED** [review]

**Review by pashiav — APPROVED** [review]
Saved without issue!
Screen Shot 2024-03-06 at 9 39 50 AM

**Review by emenslin — APPROVED** [review]
Looks good, can add a blank cell to visual editor and save with no errors.

**Pull request merged**